tox.ini

```diff
@@ -53,7 +54,8 @@ commands =
         --apply \
         --dont-skip __init__.py \
         --skip-glob pex/vendor/_vendored/** \
-        --skip-glob .pyenv_test/**
+        --skip-glob .pyenv_test/** \
+        --skip-glob .env/**
```

Comment:
Seeing this added and no corresponding `.gitignore` entry added is suspicious.

Reply:
yep, my bad, unintentional checkin of that change.